python/cli/prelude_cli/views/build.py

```diff
@@ -41,23 +41,28 @@ def create_test(controller, name, unit, test, techniques, advisory):
             advisory=advisory
         )
 
-    if not test:
-        basename = f'{t["id"]}.go'
-        template = pkg_resources.read_text(templates, 'template.go')
+    def create_template(template, name):
+        utc_time = str(datetime.now(timezone.utc))
+        template = pkg_resources.read_text(templates, template)
         template = template.replace('$ID', t['id'])
         template = template.replace('$NAME', name)
         template = template.replace('$UNIT', unit or '')
-        template = template.replace('$CREATED', str(datetime.utcnow()))
+        template = template.replace('$TIME', utc_time)
         
         with Spinner(description='Applying default template to new test'):
-            controller.upload(test_id=t['id'], filename=basename, data=template.encode('utf-8'))
-            t['attachments'] = [basename]
+            controller.upload(test_id=t['id'], filename=name, data=template.encode('utf-8'))
+            t['attachments'] += [name]
 
-        test_dir = PurePath(t['id'], basename)
+        dir = PurePath(t['id'], name)
         Path(t['id']).mkdir(parents=True, exist_ok=True)
```

Comment:
no need for this `mkdir` to be in the inner function - i'd pull it out into the main function logic